src/nvim/path.c

```diff
@@ -210,7 +210,7 @@ int path_head_length(void)
 /// @return
 ///   - True if path begins with a path head
 ///   - False otherwise
-bool is_path_head(const char_u *path)
+bool is_path_head(const char *path)
 {
 #ifdef MSWIN
   return isalpha(path[0]) && path[1] == ':';
```

Comment:
Oh, what I initially wanted to say is
```suggestion
  return isalpha((uint8_t)path[0]) && path[1] == ':';
```
Because
```
       The standards require that the argument c for these functions is either EOF or a value that is representable in the type unsigned char.  If the argument c is off type char, it must be cast to unsigned char
```